django/utils/translation/__init__.py

```diff
@@ -194,7 +194,20 @@ def check_for_language(lang_code):
 
 
 def to_locale(language):
-    return _trans.to_locale(language)
+    """Turn a language name (en-us) into a locale name (en_US)."""
+    language = language.lower()
+    parts = language.split('-')
+    try:
+        country = parts[1]
+    except IndexError:
+        return language
+    else:
```

Comment:
Unrelated to this change but this `else` branch is unnecessary given we return on `IndexError`.